packages/generator/src/v2.0/generatePerQueryDataFiles.ts

```diff
@@ -245,22 +250,22 @@ export function queryParamJsDoc(
   return ret;
 }
 
-function getObjectImports(
-  enhancedOntology: EnhancedOntologyDefinition,
-  objectTypes: string[],
-  filePath: string,
-) {
-  let ret = "";
-
-  for (const fqObjectApiName of objectTypes) {
-    const obj = enhancedOntology.requireObjectType(fqObjectApiName);
-    ret += `import type { ${obj.getDefinitionIdentifier(true)} as ${
-      obj.getImportedDefinitionIdentifier(true)
-    } } from "${obj.getImportPathRelTo(filePath)}";\n`;
-  }
-
-  return ret;
-}
+// function getObjectImports(
```

Comment:
Why are we commenting this out?

Reply:
Oops. Meant to delete that. We had two of these it turned out and I changed this to use the other one.